index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Palindrome Checker</title>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <style>
+    body {
+      font-family: Arial, sans-serif;
+      text-align: center;
+      background: linear-gradient(90deg, white, black, red);
+      color: white;
+      margin: 0;
+      padding: 20px;
+      box-sizing: border-box;
+    }
+
+    h1 {
+      color: red;
+    }
+
+    label {
+      display: block;
+      font-size: 20px;
+      margin-bottom: 10px;
+    }
+
+    input[type="text"] {
+      width: 300px;
+      height: 30px;
+      font-size: 18px;
+      padding: 5px;
+    }
+
+    button {
+      background-color: red;
+      color: white;
+      font-size: 20px;
+      padding: 10px 20px;
+      border: none;
+      cursor: pointer;
+    }
+
+    button:hover {
+      background-color: darkred;
+    }
+
+    #result {
+      font-size: 24px;
+      margin-top: 20px;
+    }
+  </style>
```

Comment:
- You have a separate file for your CSS code which is `style.css` move your styling code there.
  - Don't forget to link your CSS file in your HTML file using link tag in the head of your HTML file.

- Also your design is not responsive for mobile version please make your design responsive for the mobile version.